readme.md

```diff
@@ -1,92 +1,40 @@
 
 
-This is the Speckle Server, which coordinates communications between the various SpeckleClients.
+This is the Speckle Server, which coordinates communications between the various bits of the Speckle ecosystem.
 
+## API
 
-Speckle Server provides :
-- a basic accounts system
-- stream coordination
-- design data collation and retrieval
-- live update events
-- a basic querying mechanism
-
-## 5 Minute Docker Install ™️ (with Digital Ocean)
-
-Head over to [Digital Ocean](https://www.digitalocean.com/) and create an account. Subsequently, create a new droplet (vm) via "One Click Apps" and select "Docker". Finally, `ssh` into your new virtual machine.
-
-Execute the following commands:
-
-    $ mkdir speckle && cd speckle
-    $ curl https://raw.githubusercontent.com/speckleworks/SpeckleServer/master/docker-compose.yml -o docker-compose.yml
-    $ docker-compose up
-
-⚠️ *Imporant security notice*: Finally, make sure  you  create a digital ocean cloud firewall rule (and apply it to your droplet) to allow port 3000, or whatever port you're using. See discussion: https://github.com/speckleworks/SpeckleServer/issues/90#issuecomment-400011794
-
-![screenshot](https://user-images.githubusercontent.com/7696515/41862603-4972fcc0-789c-11e8-8a79-268280376a1a.png)
-
-Note: You're not restricted to digital ocean. Any machine with docker should be ok.
-
-## Deploying to Heroku
-
-    $ heroku create --stack cedar
-    $ heroku addons:create mongolab:sandbox
-    $ heroku addons:create heroku-redis:hobby-dev
-    $ git push heroku master
-    $ heroku open
-
-## Deploying on Debian-based OSes (Ubuntu etc)
-
-1) Install mongodb, redis servers and npm: 
-
-       $ sudo apt-get install mongodb redis npm
-
-2) If you don't want both the redis and mongo servers running all the time (For ex. if you are just testing), disable both startup scripts (If you wish to leave both running automatically, skip to step 4):
-
-       $ sudo systemctl disable mongodb
-       $ sudo systemctl disable redis-server`
-
-3) And stop both mongo and redis processes that were started automatically by apt-get: 
-
-       $ sudo systemctl stop mongodb
-       $ sudo systemctl stop redis-server
+[API docs are here](https://speckleworks.github.io/SpeckleSpecs/) - they are a good overview of what you can do.
 
-4) Clone SpeckleServer and run npm to install the needed nodejs packages: 
+## Installation
 
-       $ git clone https://github.com/speckleworks/SpeckleServer.git
-       $ cd SpeckleServer
-       $ npm install
+The Speckle Server is a nodejs (`v.8+`, `latest stable` preffered) app.
 
-5) Follow the instructions in `.env-base` file to configure your server.
-    
-6) Start mongo (create a folder somewhere to store the db): 
+- Dependencies: [mongodb](https://en.wikipedia.org/wiki/MongoDB) and [redis](https://en.wikipedia.org/wiki/Redis). 
```

Comment:
I think that these should link to the official project websites, like https://www.mongodb.com/download-center/community and https://redis.io/